### ISSUE TITLE
Support oplog tailing for this package installation

### ISSUE THREAD

**Issue by anmol26s** [YunoHost-Apps/rocketchat_ynh#37]
When I go to Administration -> Info I see this error on the top of the page.

```
Error: Rocket.Chat requires oplog tailing when running in multiple instances.
Please make sure that your MangoDB is on ReplicaSet mode and Mango_OPLOG_URL 
environment is defined correctly on the application server.
```
Though the RocketChat is working fine.

**Comment by selamanse** [comment]
yeah I noticed that message. it's there is no oplog setup included in this setup (it's not necessary from my point of view for small setups e.g. 20 people or so). But yes, it should be an option to install it to so I take this would be a feature in upcoming versions. I'm just thinking of loud. 
As of the message - it disappears after a certain time and doesn't really seem to be an issue...

**Comment by selamanse** [comment]
Will be resolved after #48
